content/apply_org.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
+
+* _Active Members_ are individuals who volunteer their time, expertise, and talent towards the work of the organization. Active members have voting rights. Membership fees for active members are waived.
+* _Supporting Members_ are individuals or organizations who support the work of the organization through their membership fees. Supporting members do not have voting rights. Annual membership fees are $50 in the US, £50 in the UK, €50 in the European Union, or the equivalent in their local currencies.
+* _Associate Members_ are noncommercial organizations whose activities are related to our mission. Associate members do not pay membership fees and do not have voting rights. 
```

Comment:
```suggestion
```

I would not collect info or speak about Associate members until we have a clear program for them.